### DISCUSSION TITLE
Add Firestore support besides Dio

### DISCUSSION THREAD

**Discussion by tyukesz** [nylo-core/nylo#89] [Ideas]
Hi there,

I'm wondering if it would be possible to add some support regarding to Firestore.
 
I would see it fitting in current networking layer besides Dio. There are a lot of cases when we want to use Firestore for accessing data and for those cases currently there isn't any clue in the documentation.

Still the behavior looks pretty much the same as in case of a HTTP request, except the fact that we have an existing package for managing the TCP communication (https://pub.dev/packages/cloud_firestore):

Fetch data -> JSON -> Map (decode) -> Model

**Comment by agordn52** [comment]
Hi @tyukesz,

That would be a great addition to the micro-framework.

A lot of mobile developers are using Firebase so this makes sense.
I'll think of some ideas.
